OOP/BankAccount.cs

```diff
@@ -0,0 +1,100 @@
+﻿namespace OOP
+{
+    public class BankAccount
+    {
+        private int _AccountNumber2 = 15674895;
+        private int _AccountNumber = 15674894;
+        private double _Balance = 15200;
+        private double _Balance2 = 17950;
+        private string _Savings = "Сберегательный счёт";
+        private string _Credit = "Кредитный счёт";
+
+
+        public int AccountNumber
+        {
+            get
+            {
+                return 15674894;
```

Comment:
Почему возвращается константное значение? Неправильно! Должно возвращаться значение поля.
```suggestion
                return _AccountNumber;
```